poetry/factory.py

```diff
@@ -57,7 +57,7 @@ def create_poetry(
         name = local_config["name"]
         version = local_config["version"]
         package = ProjectPackage(name, version, version)
-        package.root_dir = poetry_file.parent
+        package.root_dir = original_root or poetry_file.parent
```

Comment:
This will sort of defeat the purpose, since we really want the build to use the source in `cwd`. We might want to consider another approach to this one.

Reply:
So after reading #2046 and especially the fact that these relative paths work fine if you build ONLY the wheel or ONLY the sdist... I have to ask, where is the need for this "isolated source copy" coming from? Building only the sdist doesn't seem to need it, building only the wheel doesn't seem to need it. So why is the CompleteBuilder trying to do something more than simply running both individual builders?